components/ILIAS/Authentication/classes/Setup/class.ilAuthenticationDatabaseUpdateSteps8.php

```diff
@@ -38,4 +38,14 @@ public function step_1(): void
             );
         }
     }
+
+    public function step_2(): void
```

Comment:
Please create a new/dedicated update step file, because the name of the current one targets "ILIAS 8".

Suggestion: `AbandonAuthRichTextEditorDatabaseUpdateSteps`

Reply:
ok